Firebase/ios/Bonfire/Mobile/FirebaseRemoteConfig.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import Firebase
+import FirebaseRemoteConfig
```

Comment:
Weird that I'm not getting build errors - I've started doing just `import Firebase` and it seems to include everything, as the docs claim it should